services/QuillLMS/spec/mailers/previews/user_mailer_preview.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+  class UserMailerPreview < ActionMailer::Preview
+    def ell_starter_diagnostic_info_email
+      UserMailer.ell_starter_diagnostic_info_email('pk', 'pkong@quill.org')
```

Comment:
Maybe use an ENV variable instead of real email.

Reply:
It doesn't actually send an email

Reply:
My comment was more about your real email being hard coded in the github repo than it being sent.  Since it's not being sent, no need for an ENV variable.  I would put in something generic like:`user@example.com`